pandas/tests/test_strings.py

```diff
@@ -97,53 +97,6 @@ def test_iter_object_try_string(self):
         assert i == 100
         assert s == 'h'
 
-    def test_cat(self):
-        one = np.array(['a', 'a', 'b', 'b', 'c', NA], dtype=np.object_)
-        two = np.array(['a', NA, 'b', 'd', 'foo', NA], dtype=np.object_)
-
-        # single array
-        result = strings.str_cat(one)
```

Comment:
@jreback 
> I removed one test for the internal method that has been factored away

Please have a look here - this is directly importing the internal method and testing it (*not* `str.cat`)

Reply:
ok